sirepo/job_supervisor.py

```diff
@@ -295,6 +303,58 @@ def __db_write(self):
         sirepo.util.json_dump(self.db, path=self.__db_file(self.db.computeJid))
         return self
 
+    @classmethod
+    def _get_running_jobs(cls, want_adm=False):
+        def _get_jobs():
+            o = []
+            for i in filter(
+                    lambda x: x.db.status == job.RUNNING,
+                    cls.instances.values(),
+            ):
+                d = i.db
+                s = d.computeJobStart
+                l = d.lastUpdateTime
+                r = [
+                    d.simulationType,
+                    d.simulationId,
+                    _strftime(s),
+                    _strftime(l),
+                    round((l - s) / 60.0, 2),
+                    ' | '.join(sorted(d.driverDetails.values()))
+                ]
+                if want_adm:
+                    r.insert(0, d.uid)
+                else:
+                    r.insert(1, d.name)
+                o.append(r)
+                return o
+
+        def _strftime(unix_time):
+            return datetime.datetime.utcfromtimestamp(
+                int(unix_time),
+            ).strftime('%Y-%m-%d %H:%M:%S')
+
+        c = [
+            'App',
+            'Simulation id',
+            'Start (UTC)',
```

Comment:
For adm, we probably want queued time so we can observe queueing. For end users, I don't think it's very useful (there isn't a lot of queueing in general).